index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <nav class="container">
+      <div><img src="./assets/img/logo.png" alt="logo" /></div>
+      <div class="toggle">
+        <img src="./assets/img/toggle.png" alt="toggle" />
+      </div>
```

Comment:
- [ ] Both the logo and hamburger menu are clickable areas so kindly add either `a` or `button` for both 💪🏼